<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LightTaskManager</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionInitializeRepository"/>
    <addaction name="actionOpenRepository"/>
    <addaction name="actionAddTask"/>
    <addaction name="actionDeleteTask"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionInitializeRepository">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/images/images/FileNew.png</normaloff>:/images/images/FileNew.png</iconset>
   </property>
   <property name="text">
    <string>Initialize Repository</string>
   </property>
  </action>
  <action name="actionAddTask">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/images/images/Plus.png</normaloff>:/images/images/Plus.png</iconset>
   </property>
   <property name="text">
    <string>Add Task</string>
   </property>
  </action>
  <action name="actionDeleteTask">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/images/images/Minus.png</normaloff>:/images/images/Minus.png</iconset>
   </property>
   <property name="text">
    <string>Delete Task</string>
   </property>
  </action>
  <action name="actionOpenRepository">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/images/images/FileOpen.png</normaloff>:/images/images/FileOpen.png</iconset>
   </property>
   <property name="text">
    <string>Open Repository</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
